### PR TITLE
Fix optimizer docs

### PR DESCRIPTION
**Pull request by rschlussel-zz**
Sorry... didn't mean to close the other PR.  My branches got messed up.  I've addressed comments and also added @KBP-TDC's documentation.  Please review again.

### CODE REVIEW

File: presto-docs/src/main/sphinx/admin/query-performance-analysis.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Presto provides several commands and utilities to help with performance
+analysis of your queries:
+
+* The standard:doc:`../sql/explain` command displays both logical and
+  distributed query plans.
+* The :doc:`../sql/explain-analyze` command provides detailed execution-time metrics
+  such as a number of input and output rows at each stage and aggregated CPU
+  time.
+* The :doc:`web-interface` offers both a cluster-wide overview of all
+  queries as well as a detailed view for each query. It includes the overall
```
Comment (akshatnair) [inline presto-docs/src/main/sphinx/admin/query-performance-analysis.rst:14]: all the queries?

File: presto-docs/src/main/sphinx/admin/query-performance-analysis.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  query metrics, stage-level progress and the task-level details. The Live Plan
```
Comment (akshatnair) [inline presto-docs/src/main/sphinx/admin/query-performance-analysis.rst]: maybe put it as `Live Plan`

File: presto-docs/src/main/sphinx/admin/query-performance-analysis.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  view displays the graphical representation of a distributed query plan with
+  the execution metrics updated while a query is running.
+
```
Comment (akshatnair) [inline presto-docs/src/main/sphinx/admin/query-performance-analysis.rst:18]: extra line

File: presto-docs/src/main/sphinx/optimizer/join-distribution-type.rst [hunk 4]
```diff
@@ -1,37 +1,61 @@
...
+       ``small-table-coefficient`` or per session using the session property
+       ``small_table_coefficient``.
+
+       .. code-block:: sql
+
+          SET SESSION small_table_coefficient = 0.05;
+
+       The coefficient is there to ensure that queries do not exceed
+       ``query.max-memory-per-node``, even when they contain multiple joins.
+       The property should be set to prevent memory errors for the most complex
```
Comment (akshatnair) [inline presto-docs/src/main/sphinx/optimizer/join-distribution-type.rst]: out of memory errors

**Review by akshatnair — COMMENTED** [review]
LGTM mod few comments

File: presto-docs/src/main/sphinx/admin/query-performance-analysis.rst (hunk 1 above)
Comment (rschlussel-zz) [reply presto-docs/src/main/sphinx/admin/query-performance-analysis.rst:14]: not needed

File: presto-docs/src/main/sphinx/admin/query-performance-analysis.rst (hunk 2 above)
Comment (rschlussel-zz) [reply presto-docs/src/main/sphinx/admin/query-performance-analysis.rst]: done

File: presto-docs/src/main/sphinx/admin/query-performance-analysis.rst (hunk 3 above)
Comment (rschlussel-zz) [reply presto-docs/src/main/sphinx/admin/query-performance-analysis.rst:18]: fixed

**Review by kbajda — APPROVED** [review]

**Comment by rschlussel-zz** [comment]
merged

**Pull request closed without merging**